tasks.php

```diff
@@ -166,117 +166,117 @@
-$browser_array = [
+const TASK_BROWSERS = [
     0 => "All",
-    1 => "Internet Explorer 6.x",
-    2 => "Netscape 6.x",
-    3 => "Internet Explorer 5.x",
-    4 => "Netscape 7.x",
-    5 => "Netscape 3.x",
-    6 => "Netscape 4.x",
-    7 => "Opera",
-    8 => "Netscape 5.x",
-    9 => "Internet Explorer 4.x",
-    10 => "Lynx",
     11 => "Avant Browser",
-    12 => "Netscape 2.x",
-    13 => "Slimbrowser",
+    52 => "Chrome / Chromium",
+    27 => "Galeon",
     14 => "Interarchy",
-    15 => "Straw",
-    16 => "MSN TV",
-    17 => "Mozilla 1.4",
-    18 => "Mozilla 1.5",
-    19 => "Mozilla 1.6",
-    20 => "Mozilla Firefox 0.6",
-    22 => "Mozilla Firefox 0.7",
+    29 => "Internet Explorer 3.x",
+    9 => "Internet Explorer 4.x",
+    3 => "Internet Explorer 5.x",
+    1 => "Internet Explorer 6.x",
+    42 => "Internet Explorer 7.x",
+    48 => "Internet Explorer 8.x",
+    10 => "Lynx",
+    28 => "Konquerer",
+    51 => "Microsoft Edge",
     23 => "Mozilla 1.1",
     24 => "Mozilla 1.2",
     25 => "Mozilla 1.3",
-    26 => "Safari",
-    27 => "Galeon",
-    28 => "Konquerer",
-    29 => "Internet Explorer 3.x",
+    17 => "Mozilla 1.4",
+    18 => "Mozilla 1.5",
+    19 => "Mozilla 1.6",
     30 => "Mozilla 1.7",
     31 => "Mozilla 1.8",
+    37 => "Mozilla Camino 0.7",
+    38 => "Mozilla Camino 0.8.x",
+    43 => "Mozilla Camino 1.x",
+    36 => "Mozilla Firefox 1.0",
+    20 => "Mozilla Firefox 0.6",
+    22 => "Mozilla Firefox 0.7",
     32 => "Mozilla Firefox 0.8",
     33 => "Mozilla Firefox 0.9",
+    41 => "Mozilla Firefox 1.5.x",
+    45 => "Mozilla Firefox 2.x",
+    46 => "Mozilla Firefox 3.x",
+    16 => "MSN TV",
+    12 => "Netscape 2.x",
+    2 => "Netscape 6.x",
+    4 => "Netscape 7.x",
+    5 => "Netscape 3.x",
+    6 => "Netscape 4.x",
+    8 => "Netscape 5.x",
+    7 => "Opera",
+    50 => "Opera 10.x",
     34 => "Opera 6.x",
     35 => "Opera 7.x",
-    36 => "Mozilla Firefox 1.0",
-    37 => "Mozilla Camino 0.7",
-    38 => "Mozilla Camino 0.8.x",
     39 => "Opera 8.x",
     40 => "Opera 9.x",
-    41 => "Mozilla Firefox 1.5.x",
-    42 => "Internet Explorer 7.x",
-    43 => "Mozilla Camino 1.x",
+    99 => "Other",
+    26 => "Safari",
     44 => "Safari 2.x",
-    45 => "Mozilla Firefox 2.x",
-    46 => "Mozilla Firefox 3.x",
     47 => "Safari 3.x",
-    48 => "Internet Explorer 8.x",
     49 => "Safari 4.x",
-    50 => "Opera 10.x",
-    51 => "Microsoft Edge",
-    52 => "Chrome / Chromium",
-    99 => "Other",
+    13 => "Slimbrowser",
+    15 => "Straw",
+    // max 99
 ];
-asort($browser_array);
-$tasks_close_array = [
-    1 => "Not a Bug",
-    2 => "Won't Fix",
-    3 => "Won't Implement",
-    4 => "Works for Me",
-    5 => "Duplicate",
+const TASK_CLOSE_REASON = [
     6 => "Deferred",
+    5 => "Duplicate",
     7 => "Fixed",
     8 => "Implemented",
     9 => "Resolved",
+    1 => "Not a Bug",
+    2 => "Won't Fix",
+    3 => "Won't Implement",
+    4 => "Works for Me",
+    // max 8
```

Comment:
Highest key value is 9 here
```suggestion
    // max 9
```